pkg/util/k8s/labels_test.go

```diff
@@ -0,0 +1,33 @@
+package k8s_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/util/k8s"
+)
+
+var _ = Describe("StringToLabel", func() {
+	Context("when given a very long string", func() {
+		s := "Avalanches_of water in the midst of a calm/And the distances cataracting toward the abyss!"
```

Comment:
Did you mean "Avalanches:of"?, because otherwise I don't think anything will be replaced...